game-of-life-multiplatform/gradle.properties

```diff
@@ -16,4 +16,5 @@
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
 
-INCLUDE_ANDROID=true
\ No newline at end of file
+INCLUDE_ANDROID=true
+konan.version=0.6-dev-1005
```

Comment:
This hardcodes the kotlin native compiler to a dev version which supports kotlin multiplatform projects. Basically it adds support for `expected` and `actual` declared kotlin classes.